masterlist.yaml

```diff
@@ -15906,16 +15924,18 @@ plugins:
     clean:
       - crc: 0xB4C1E773
         util: 'SSEEdit v4.0.3'
-  - name: 'ClockworkTitleRemover.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+  - name: 'ClockworkSuperiorSorting.esp'
+    req: [ *PapyrusExtender, *SKSE ]
+    msg:
+      - <<: *scriptExtenderRequiredScripts
     clean:
-      - crc: 0x913191E9
+      - crc: 0x80123223
         util: 'SSEEdit v4.0.3'
-  - name: 'ClockworkMausoleumDummyDoorFix.esp'
+  - name: 'ClockworkTitleRemover.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
     clean:
-      - crc: 0xCAE2B6AE
-        util: 'SSEEdit v4.0.3g'
+      - crc: 0xBF1FBE1F
+        util: 'Human Review in SSEEdit v4.0.3' # It doesn't like my deleted records, but I can veryfiy that they don't cause any issues in-game.
```

Comment:
For plugins that a mod author doesn't want cleaning data displayed, we instead omit the cleaning data & leave a comment like `#Do not clean` (see https://github.com/loot/loot.github.io/issues/89 for our rationale & discussion on the topic). However, that's really only for intentional ITMs (which we prefer mod authors don't use in the first place, hence the compromise). AFAICT, deleted references should always be cleaned. I'm not aware of any rationale for not cleaning them as undeleting & disabling has the same effect as deleting without causing the issues that deleted references cause (see this brief description of [deleted references](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#DeletedReference) in the xEdit docs). They may not have caused issues in your testing as none of your other mods interacted with those references, but it will cause issues for users with plugins that do. Even if that's extremely unlikely, we would still consider that reason enough for the mod to be cleaned (or, at the very least, not to tell users otherwise).

I can't replace the cleaning data through a suggestion due to the way GitHub handles lines deletions, so you'll have to either replace it manually or remove this cleaning data if you really don't want to submit it yourself (if you remove it, you'll also have to remove *ClockworkTitleRemover.esp*'s entry as a location alone isn't enough metadata to warrant an entry).

Reply:
i'm not sure if it works, but I'm trying out github.dev on mobile (away from my desktop ATM). Bit of a pain to use but it's better than the main site crashing my mobile browser almost every time.
Let's see, shall we?